tests/observability/metrics/conftest.py

```diff
@@ -1047,3 +1055,27 @@ def deleted_ssp_operator_pod(admin_client, hco_namespace):
 @pytest.fixture(scope="class")
 def initiate_metric_value(request, prometheus):
     return get_metrics_value(prometheus=prometheus, metrics_name=request.param)
+
+
+@pytest.fixture()
+def vm_for_vm_disk_allocation_size_test(namespace, admin_client, golden_images_namespace):
+    with VirtualMachineForTests(
+        client=admin_client,
```

Comment:
why admin needed here? unpriv client should have all needed rights for this test, no?